src/Authenticator/Result.php

```diff
@@ -74,13 +74,13 @@ public function __construct($data, $code, array $messages = [])
      */
     public function isValid()
     {
-        return $this->_code > 0;
+        return strpos($this->_code, 'SUCCESS') === 0;
```

Comment:
`return $this->_code === ResultInterface::SUCCESS;`

Reply:
If you are certain that there is only one kind of SUCESS (not a range) in future.
I will adapt it for now as for now there is not.